src/Main.js

```diff
@@ -1,12 +1,35 @@
 import { Component } from 'react';
 import HornedBeast from './HornedBeast'
 
+
```

Comment:
Hi there Emily, good work so far on this! 👍 

You can go ahead and import the data json file here: import data from 'path to file here';